sdf/deployment.mdx

```diff
@@ -1,75 +1,68 @@
+Workers must be registered before deploying a dataflow. The CLI provides commands to manage workers, including creating, listing, switching, and deleting them.
 
-Once you have created the worker, You can list them:
+To list registered workers, you can use `list` command.  It will show the current selected worker with `*`.
 
 ```bash
-$> sdf worker create main
-Worker `main` created for cluster: `local`
 $> sdf worker list
-    NAME  TYPE  CLUSTER  WORKER ID                            
- *  main  Host  local    7fd7eda3-2738-41ef-8edc-9f04e500b919
+    NAME          TYPE    CLUSTER       WORKER ID                            
+ *  worker1       Remote  cluster1      afd1-2738-41ef-8edc-9f04e500b919
+    worker2       Remote  cluster1      wer2-qwerty-dummy-id
+    edge1         Remote  cluster1      edg1-worker-id
 ```
-The `*` indicates the current selected worker.  The worker id is internal unique identifier for current fluvio cluster.  Unless specified, it will be generated by the system.
 
-SDF only support running a single HOST worker for each machine since a single worker can support many dataflow.  If you try to create another worker, you will get an error message.
+InfinyOn Cloud workers are automatically provisioned and registered in your profile. All other workers must be manually registered or created using the CLI.
 
-```bash
-$ sdf worker create main2
-$ Starting worker: main2
-There is already a host worker with pid 20686 running.  Please terminate it first
-```
+To switch the worker, you can use `switch` command:
 
-Shutting down worker will terminate all running dataflow and worker process. 
 ```bash
-$> sdf worker shutdown main
-sdf worker shutdown main
-Shutting down pid: 20688
-Shutting down pid: 20686
-Host worker: main has been shutdown
+$> sdf worker switch <worker-name>
 ```
 
-Even though host worker is shutdown and removed from the profile, the dataflow files and state are still persisted.   You can restart the worker and the dataflow will resume.
+This assumes worker-name is already created or registered.  
 
-For example, if you have dataflow `fraud-detector` and `car-processor` running in the worker and you shut down the worker, the dataflow process will be terminated.   But you can resume by recreating the HOST worker.
 
-```bash
-$> sdf worker create main
-```
 
-The local worker store the dataflow state in the local file system.  The dataflow state is stored in the `~/.sdf/<cluster>/worker/<dataflow>`.
-Sor for `local` cluster, files will be stored in `~/.sdf/local/worker/dataflows`.
+##  Worker types
 
-if you have delete the fluvio cluster, worker need to be manually shutdown and created again.  This limitation will be removed in the future release
+There are two types of workers: `Host` and `Remote`.  `Host` is worker that use a simple deployment model suitable for local machine or edge device.  It is not designed for robust production deployment.  SDF CLI is used to manage host worker.
```

Comment:
```suggestion
There are two types of workers: `Host` and `Remote`.  A `Host` is worker that uses a simple deployment model suitable for a local machine or edge device.  It is not designed for robust production deployment.  The SDF CLI is used to manage a host worker.
```

Reply:
It might be worth saying that `Host` is for the `run` command and `Remote` is for the `deploy` command here

Reply:
Not true.  `Host` and `Remote` are types of workers.   `run` doesn't interact with workers